.github/workflows/check_entrypoint.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow is responsible for verifying the standard library with Kani.
```

Comment:
I'm confused. What is the difference between this is kani.yml

Reply:
Deleted the second workflow and moved the new steps to kani.yml. I wanted to have a check just for the script itself, but this is probably simpler.